wdio/specs/edit.js

```diff
@@ -0,0 +1,47 @@
+const assert = require('assert');
+
+describe('edit page', () => {
+
+  const loginWithUserNoam = () => {
```

Comment:
this should be added via `browser.addComand` like [here](https://github.com/Gizra/circuit-bid/blob/cc52cdaafa0f53df81cfd02bae05f73df66b767a/client/test/config/custom-commands.js#L6-L20)